demonstrations/tutorial_diffable_shadows.py

```diff
@@ -3,7 +3,7 @@
 
 .. meta::
     :property="og:description": Classical shadows in the Pauli basis
-    :property="og:image": https://pennylane.ai/qml/_images/diffable_shadows/thumb.png
+    :property="og:image": https://pennylane.ai/qml/_images/diffable_shadows/pauli_shadows.jpg
```

Comment:
ahhh @KetpuntoG you merged before I could comment 😆 There is a typo, it should be
```suggestion
    :property="og:image": https://pennylane.ai/qml/_images/pauli_shadows.jpg
```

You can see by clicking the link: https://pennylane.ai/qml/_images/pauli_shadows.jpg

Reply:
oh nooo, okey, another PR 😆